### DISCUSSION TITLE
Error while using subapis at mounted swagger js and css

### DISCUSSION THREAD

**Discussion by rodrigoga799** [fastapi/fastapi#7276] [Questions]
I have use the documentation for mounting the swagger docs as https://fastapi.tiangolo.com/advanced/extending-openapi/
but when I try to mount using this 

authAPI = FastAPI(docs_url=None, redoc_url=None)

@authAPI.get("/docs", include_in_schema=False)
app.mount("/api/v1.0/auth" , authAPI , name="authAPI")
async def custom_swagger_ui_html():
    return get_swagger_ui_html(
        openapi_url=authAPI.openapi_url,
        title=authAPI.title + " ENTEL SGIMP ",
        oauth2_redirect_url=app.swagger_ui_oauth2_redirect_url,
        swagger_js_url="/static/swagger-ui-bundle.js",
        swagger_css_url="/static/swagger-ui.css",
    )

the authAPI docs shows are from the openapi.json file not from the authAPI.Why does have this behavior?
Is there any way to fix it ? 

[main.zip](https://github.com/tiangolo/fastapi/files/5165215/main.zip)

**Comment by ycd** [comment]
@rodrigoga799 feel free to edit the question with a proper code formatting, and I do not know what is in the main.zip but if it is main.py and it is too long to paste here, create a gist for it. So others can help with ease.

**Comment by tiangolo** [comment]
Thanks for the help @ycd! 🍰 

@rodrigoga799 I'll close this issue now, but if you are still having problems, please create a new issue and follow the template form. That will allow us to properly understand your question and help you.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
